base/applications/iexplore/iexplore.inf

```diff
@@ -22,7 +22,7 @@ HKCR,"CLSID\%CLSID_Internet%","LocalizedString",0x00020000,"@%SystemRoot%\system
 HKCR,"CLSID\%CLSID_Internet%\DefaultIcon",,0x00020000,"%SystemRoot%\system32\shell32.dll,-14"
 HKCR,"CLSID\%CLSID_Internet%\Shell",,,"open"
 HKCR,"CLSID\%CLSID_Internet%\Shell\open",,,""
-HKCR,"CLSID\%CLSID_Internet%\Shell\open\Command",,,"rundll32.exe url,OpenURL https://reactos.org/"
+HKCR,"CLSID\%CLSID_Internet%\Shell\open\Command",,,"rundll32.exe url,OpenURL https://www.google.com/"
```

Comment:
Well... why not another search engine? It would actually be better to just use the `about::blank` default page, so that there is no imposed choice of the search engine on the user.

Reply:
Ideally it should open the homepage set in the Internet control panel options.

Reply:
`https://` (only this) is preferred but ReactOS has a bug in opening `https://` (only this):

https://jira.reactos.org/browse/CORE-19042

Reply:
`about:blank` is not applicable in this case.

Reply:
I absolutely love the proposal with about:blank. Why isn't it applicable in this case? I do not understand yet.

Reply:
`about:blank` is not a valid URL.

Reply:
I can confirm that I cannot open
`rundll32.exe url, OpenURL about:blank` from the rundlg of XPSP3 with the aim to open IE.
But `rundll32.exe url, OpenURL about:blank https://google.com` works.
So maybe this is enough justification. Thx.
A pity, I would have liked an empty browser tab very much.
I would vote then for either "the homepage set in the Internet control panel options" or `https://google.com`
I do think both is more helpful here for the user than `https://reactos.org`

Reply:
Used `https://www.google.com`.

Reply:
Not everyone is fond of google by the way (e.g. in France).
https://alternativeto.net/software/google-search/?license=opensource

Reply:
I have proposed a revert below...

Reply:
@JoachimHenze about:blank is not an URI, but a browser magic string.